mod/03_machines/lib/javascript/wagn.js.coffee

```diff
@@ -150,6 +150,9 @@ $(window).ready ->
   $('body').on 'ajax:success', '.slotter', (event, data, c, d) ->
     unless event.slotSuccessful
       $(this).slotSuccess data
+      # should scroll to top after clicking on new page
+      if $(this).hasClass "card-paging-link"
+        $("body").scrollTop 0
```

Comment:
Does this apply to next and prev?  Should it go to bottom for prev?  Does it already?